### PR TITLE
Implement the nudge for symptom reports

### PR DESCRIPTION
**Pull request by lemonlatte**
In this PR, it implements the nudge mechanism which is defined in https://github.com/bitmark-inc/autonomy/issues/37.

## Changes

- Introduce a worker to handle all nudge related jobs.
- Add a new function to send notification by give texts
- Add a function to generate notification body which support two languages (en, zh_tw)
- Add a workflow to check this nudge for every accounts

## Workflow description

- The workflow contains two parts:
  - It first check if there is a report in previous day. If there is, it checks whether a notification has sent according to logic defined in https://github.com/bitmark-inc/autonomy/issues/37. 
  - When the condition matches, it will send notification to the related account. 
- The workflow will be triggered when a new report is submitted. 
- It will persist for 24 hours and check the above condition for every hour.

### CODE REVIEW

File: api/symptom.go [hunk 1]
```diff
@@ -151,6 +152,14 @@ func (s *Server) reportSymptoms(c *gin.Context) {
 				sentry.CaptureException(err)
 			}
 		}()
+
+		go func() {
```
Comment (pieceofr) [inline api/symptom.go:156]: It isn't utils.TriggerAccountSymptomFollowUpNudge synchronized ? Why do we need a goroutine ?

Comment (lemonlatte) [reply api/symptom.go:156]: Just not to block this API. Same to `TriggerPOIUpdate` above.

Comment (pieceofr) [reply api/symptom.go:156]: Should other triggers add a goroutine?  The last discussion is that behavior/symptom triggers are synchronized so ... is these two asynchronized?

Comment (lemonlatte) [reply api/symptom.go:156]: I do not know clearly about the definition of asynchronized you mentioned here. The job is invoked in the background. The cadence api call to invoke the job happens in the `reportSymptoms` API.

**Pull request merged**